Makefile.am

```
@@ -47,6 +47,9 @@ EXTRA_DIST = \
 
 # coverage
 
+CODE_COVERAGE_LCOV_RMOPTS = \
```

"so ignore the error" -> "to ignore the error"?

---

Thanks! fixed that and will set MWP.